src/Data/Bin/Base.agda

```diff
@@ -0,0 +1,109 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Arithmetic of binary represented natural numbers. Initial part.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Bin.Base where
+
+open import Algebra.FunctionProperties using (Op₂)
+open import Data.Nat        using (ℕ) renaming (suc to 1+_; _*_ to _*ₙ_)
+open import Function        using (_∘_)
+open import Relation.Binary using (Decidable)
+open import Relation.Binary.PropositionalEquality using (_≡_; refl; cong)
+open import Relation.Nullary                      using (yes; no)
+
+
+data Bin : Set
+  where
```

Comment:
Please put the `where` on the same line as the `Set`.

Reply:
Done